cmd/access.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewAccessCommand creates a new `qri apply` cobra command for applying transformations
```

Comment:
Change to "NewAccessCommand creates a new `qri access` cobra command for managing permissions"